lib/utils.js

```diff
@@ -13,6 +13,25 @@ module.exports = {
     });
   },
 
+	getApiGatewayStageName() {
+		let stageNameStage, stageNameDeploy;
+		const serviceTemplate = this.serverless.service.resources;
+		Object.keys(serviceTemplate.Resources).forEach(function(key){
+			if (serviceTemplate.Resources[key]['Type'] == 'AWS::ApiGateway::Stage') {
+				stageNameStage = serviceTemplate.Resources[key].Properties.StageName;
```

Comment:
This assumes the last ApiGateway::Stage to be iterated over is the correct one, rather you should determine if its the one being deploye by ApiGatewayDeployement.